controlplane/src/db/schema.ts

```diff
@@ -964,6 +998,10 @@ export const auditLogs = pgTable('audit_logs', {
   targetType: text('target_type'), // the type of the target e.g. organization
   targetDisplayName: text('target_display_name'), // human-readable name of the target e.g. organization name
 
+  // Namespace information
+  targetNamespaceId: text('target_namespace_id'), // The id of the namespace in which the action is performed
```

Comment:
What will happen when the information is missing. Do we need to migrate it?

Reply:
It will just not show anything in the UI. nothing to do here

Reply:
Update migration to populate existing audit logs